feature-odyssey/src/main/kotlin/com/hedvig/android/odyssey/navigation/ClaimFlowGraph.kt

```diff
@@ -144,9 +149,11 @@ internal fun NavGraphBuilder.claimFlowGraph(
         imageLoader = imageLoader,
       )
     }
-    animatedComposable<ClaimFlowDestination.SingleItemPayout> {
-      ClaimFlowBackHandler(navController, finishClaimFlow)
-      SingleItemPayoutDestination()
+    animatedComposable<ClaimFlowDestination.ClaimSuccess> {
+      BackHandler { finishClaimFlow() }
+      // todo backend for success returns nothing but an ID. Not sure if it's only missing on the backend or if it's a
+      //  different screen.
+      ClaimSuccessDestination()
```

Comment:
Okay thanks for that. It's because I didn't find any design for this one in particular in Figma, only saw designs for when we also get the number back and can do a payout and so on, but those flow steps are not implemented yet afaik.
Gonna look at how this looked like in the old flow since as I understand it's the exact same screen if I don't find any figma screen which I missed